pkg/clusterresource/builder.go

```diff
@@ -156,6 +159,9 @@ func (o *Builder) Build() ([]runtime.Object, error) {
 	var allObjects []runtime.Object
 	allObjects = append(allObjects, o.generateClusterDeployment())
 	allObjects = append(allObjects, o.generateMachinePool())
+	if !o.SkipMachinePoolGeneration {
```

Comment:
Hey, I hit you up on slack about the machinepool missing stuff in `platform`. 

Is this why the `platform` part of the machinepool wasn't filled out?

Here's what it looked like:
```yaml
apiVersion: v1
items:
- apiVersion: hive.openshift.io/v1
  kind: MachinePool
  metadata:
    annotations:
      kubectl.kubernetes.io/last-applied-configuration: |
        {"apiVersion":"hive.openshift.io/v1","kind":"MachinePool","metadata":{"annotations":{},"name":"abutcher2-worker","namespace":"default"},"spec":{"clusterDeploymentRef":{"name":"abutcher2"},"name":"worker","platform":{"ovirt":{}},"replicas":3},"status":{}}
    creationTimestamp: "2020-06-29T14:37:15Z"
    finalizers:
    - hive.openshift.io/remotemachineset
    generation: 1
    name: abutcher2-worker
    namespace: default
    resourceVersion: "29035149"
    selfLink: /apis/hive.openshift.io/v1/namespaces/default/machinepools/abutcher2-worker
    uid: 98cd33d3-66e4-4220-a354-700e9d46ab50
  spec:
    clusterDeploymentRef:
      name: abutcher2
    name: worker
    platform:
      ovirt: {}
    replicas: 3
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
```

Reply:
I think your PR should remove this.

Reply:
Ok, will do.

Just wondering, though, the line above the if statement is exactly the same as the line inside the if statement, so I think that means that you're always generating the `machinepool`.